### PR TITLE
Pop up modal window

### PR DESCRIPTION
**Pull request by samiullahbahadur**
# We used JavaScript to implement modal popup window.

 1. I create modal section [here.](https://github.com/samiullahbahadur/Portfolio/blob/d4215e1d6f2121568876273d9c2c09ebf61e81fd/index.html#L237-L286)
 2.  I create popup window in JavaScript [here](https://github.com/samiullahbahadur/Portfolio/blob/d4215e1d6f2121568876273d9c2c09ebf61e81fd/js/script.js#L54-L128)
 
 3. I stored title, content and links in JavaScript object and array [here](https://github.com/samiullahbahadur/Portfolio/blob/d4215e1d6f2121568876273d9c2c09ebf61e81fd/js/script.js#L31-L52)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -83,76 +83,76 @@ <h4>Multi-Post Stories</h4>
         
             <div class="cards-section">
                 <div class="work-card">
-                    <div class="card-content">
-                        <h4>Professional Art Printing Data</h4>
+                    <div class="card-content flex-container " >
+                        <h4 class="title" > 1st: Printing Data</h4>
```
Comment (moise10r) [inline index.html:87]: https://github.com/samiullahbahadur/Portfolio/blob/d4215e1d6f2121568876273d9c2c09ebf61e81fd/index.html#L70-L158


- [ ] As required :point_down: 

image

- [ ] As seen above, all the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.

This means you can create an array of objects to store the `card-content `project information then populate the `work-card` section of the main page entirely with javascript!

If you are struggling or don't know how to do so, here is an [example](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)

 - All those cards information should be stored into an array of objects then create a function to render through the array and create dynamically your HTML with the information from the object array, the works section must stay empty

**Review by moise10r — CHANGES_REQUESTED** [review]
### Hi **@samiullahbahadur,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- The popup is responsive, it fits mobile and desktop sizes properly.✔️ 
- You have a professional README file
-  You have a descriptive PR, keep rocking :rocket: 
 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (samiullahbahadur) [reply index.html:87]: I'm did it before, You should checked my  JavaScript code and then suggest for me, I think you checked just index page you don't checked my js code
what do you mean man?
you should connect with me by zoom and I will show you this parts.

File: index.html [hunk 2]
```diff
@@ -230,6 +230,60 @@ <h6 id="text">I'm always interested in hearing about new projects, so if you'd l
...
+                          src="image/github.png"
+                          alt="git hub icon"
+                        />
+                      </button>
+                    </li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+          </section><!-- #modal section -->
```
Comment (Hamzaoutdoors) [inline index.html:286]: - [OPTIONAL]
 Great work making the modal popup showing up in your project, but as you can see in the screenshot below you need to create the modal window dynamically using JavaScript array. Each object contains at least the following information about the project: name, description, featured image, technologies, link to` live version`, and link to the project `source code`. i.e. : 

```JS
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
Capture

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @samiullahbahadur, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 
- [x] Pop window behaviors working good ✔️ 


_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (moise10r) [reply index.html:87]: Hi @samiullahbahadur, 

https://github.com/samiullahbahadur/Portfolio/blob/463880b6331577c1b02b42227f5dee2503cadf54/js/script.js#L1-L235



I checked your javascript file but couldn't find an array of objects with information for your map and popups as you can see above :arrow_up: , please make the necessary changes

**Pull request merged**